ProcessMaker/Http/Controllers/Designer/DesignerController.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace ProcessMaker\Http\Controllers\Designer;
+
+use Illuminate\Auth\Access\AuthorizationException;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+use ProcessMaker\Http\Controllers\Controller;
+use ProcessMaker\Traits\HasControllerAddons;
+
+class DesignerController extends Controller
```

Comment:
This New Controller has been created but can't be tested on local environment before Push the changes because there are problems with new versions of PHP. A new sub task was created to solve this request: https://processmaker.atlassian.net/browse/FOUR-10279